package.json

```diff
@@ -5,6 +5,9 @@
   "author": "Discourse",
   "license": "MIT",
   "devDependencies": {
-    "eslint-config-discourse": "^1.1.8"
+  "eslint-config-discourse": "^1.1.9"
+  },
+  "dependencies": {
+    "event-from": "^1.0.0"
```

Comment:
As I understand, this isn't used and only points toward the source of the vendored file?

Reply:
Yes, it is just the source of the vendored file in `assets/javascripts/discourse/lib/event-from.js.es6`.